cluster/router/tag/tag_router.go

```diff
@@ -55,6 +65,16 @@ func (c *tagRouter) isEnabled() bool {
 	return c.enabled
 }
 
+func (c *tagRouter) SetApplication(app string) {
+	c.application = app
+}
+
+func (c *tagRouter) tagRouterRuleCopy() RouterRule {
+	fmt.Println(c.tagRouterRule, "fuck")
```

Comment:
```suggestion
	fmt.Println(c.tagRouterRule, "fuck")
```
i think this feature has made you crazy ：）